test/tst_netcdftime.py

```diff
@@ -246,7 +246,7 @@ def runTest(self):
         d2 = datetime(1995, 1, 1)
         d3 = datetimex(2001, 2, 30)
         assert hash(d1) == hash(d1)
-        assert hash(d1) == hash(d2)
+        assert hash(d1) != hash(d2) # represent the same date, but are *not* the same object
```

Comment:
See https://docs.python.org/2/reference/datamodel.html#object.__hash__

"The only required property is that objects which compare equal have the same hash value"